Index: pages/index.test.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import React from 'react';\r\nimport { render, screen } from '@testing-library/react';\r\nimport Home from './index';\r\n\r\ndescribe('jest test', () => {\r\n  it('test', () => {\r\n    render(<Home />);\r\n    expect(screen.getByText('Welcome to')).toBeInTheDocument();\r\n  });\r\n});\r\n
===================================================================
diff --git a/pages/index.test.tsx b/pages/index.test.tsx
--- a/pages/index.test.tsx	
+++ b/pages/index.test.tsx	
@@ -1,10 +1,9 @@
-import React from 'react';
 import { render, screen } from '@testing-library/react';
-import Home from './index';
+import Home from '@pages/index';
 
-describe('jest test', () => {
-  it('test', () => {
+describe('home', () => {
+  it('render', () => {
     render(<Home />);
-    expect(screen.getByText('Welcome to')).toBeInTheDocument();
+    expect(screen.getByRole('heading', { name: /welcome to next\.js!/i }));
   });
 });
