Index: pages/api/hello.test.ts
===================================================================
diff --git a/pages/api/hello.test.ts b/pages/api/hello.test.ts
new file mode 100644
--- /dev/null	
+++ b/pages/api/hello.test.ts	
@@ -0,0 +1,19 @@
+import { createMocks } from 'node-mocks-http';
+import handler from '@pages/api/hello';
+
+describe('/api/hello', () => {
+  it('api', async () => {
+    const { req, res } = createMocks({
+      method: 'GET',
+    });
+
+    await handler(req, res);
+
+    expect(res._getStatusCode()).toBe(200);
+    expect(JSON.parse(res._getData())).toEqual(
+      expect.objectContaining({
+        name: 'John Doe',
+      })
+    );
+  });
+});
